prql-compiler/src/sql/dialect.rs

```diff
@@ -81,11 +81,12 @@ impl Dialect {
             | Dialect::SQLite
             | Dialect::Postgres
             | Dialect::MySql
-            | Dialect::MsSql => SupportLevel::Supported,
+            | Dialect::MsSql
+            | Dialect::ClickHouse => SupportLevel::Supported,
```

Comment:
Depending on how many exclusions we need, we could move this down a rung. (Probably we should do that for MsSql too)